src/main/flight/pid_init.c

```diff
@@ -272,6 +274,8 @@ void pidUpdateFeedforwardLpf(uint16_t filterCutoff)
     if (filterCutoff > 0) {
         for (int axis = FD_ROLL; axis <= FD_YAW; axis++) {
             pt3FilterUpdateCutoff(&pidRuntime.feedforwardPt3[axis], pt3FilterGain(filterCutoff, pidRuntime.dT));
+            pt3FilterUpdateCutoff(&pidRuntime.angleFeedforwardPt3[axis], pt3FilterGain(filterCutoff, pidRuntime.dT));
```

Comment:
It's probably that we could / should over-filter harder here as well as the PT2.
There are some residual timing glitches that we could eliminate if this was perhaps set 2x or 4x stronger.

Reply:
I've now done quite a lot of testing on this.
With strong PT3 filtering we can not worry about the PT2.
I found that using 1/20th the normal PT3 cutoff worked really well.  Very smooth angleFeedforward traces, reduced motor spiking, and not much worse overall lag, depending on how it is evaluated.

Reply:
I've suggested the changes needed to make the PT3 smoothing adjustable